docling/backend/genos_hwp_backend.py

```
@@ -0,0 +1,564 @@
+                page_no=1, 
+                size=Size(width=595, height=842) # 표준 A4 pt
+            )
+
+    @override
+    def unload(self):
+        """추상 메서드 구현: 리소스 해제"""
+        if self.temp_input_path and self.temp_input_path.exists():
+            try:
+                os.remove(self.temp_input_path)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)